### DISCUSSION TITLE
New type icon stile vs old

### DISCUSSION THREAD

**Discussion by roguedex-dev** [roguedex-dev/roguedex#26] [Polls]
https://github.com/roguedex-dev/roguedex/pull/22

![329716397-0269e0ae-9559-4ef2-9ab2-6ef610a9b0bf](https://github.com/roguedex-dev/roguedex/assets/169194072/1b8a3b22-ffb1-4181-b842-7656d33a4bae)
![329716423-1c5575a0-39ac-4097-bffe-73bc746a3252](https://github.com/roguedex-dev/roguedex/assets/169194072/ff4c5dfe-f870-443e-a5c1-db312a3cd662)

**Comment by paulobatista** [comment]
Is the "Modern" type icon style the first one?
